src/crypto/index.ts

```diff
@@ -1426,7 +1426,11 @@ export class Crypto extends TypedEventEmitter<CryptoEvent, CryptoEventHandlerMap
     /*
      * Event handler for DeviceList's userNewDevices event
      */
-    private onDeviceListUserCrossSigningUpdated = async (userId: string) => {
+    private onDeviceListUserCrossSigningUpdated = (userId: string): void => {
+        // TODO: why are we not handling errors?
+        void this.onDeviceListUserCrossSigningUpdatedPromise(userId);
+    };
+    private onDeviceListUserCrossSigningUpdatedPromise = async (userId: string) => {
```

Comment:
I think naming a function `...Promise` is misleading

Reply:
I've committed an alternative way to do this.